crm_request/data/holiday_closure_email_template.xml

```diff
@@ -12,23 +11,19 @@
             <field name="body_html" type="html">
                 <div>
                     % set claim = object
-                    % set today = datetime.date.today().strftime('%Y-%m-%d')
-                    % set closure = object.env["holiday.closure"].search([('start_date', '&lt;=', today),('end_date', '&gt;=', today)], limit=1)
-                    % set date_start = closure.get_date('start_date')
-                    % set date_end = closure.get_date('end_date')
```

Comment:
You should keep this, because the method `get_date` will format the date in the language of the partner and will display it better than directly accessing it which will display 2019-05-23.